output/table.go

```diff
@@ -2,14 +2,13 @@ package output
 
 import (
 	"fmt"
+	"io"
 	"strings"
 	"text/tabwriter"
-
-	"github.com/codegangsta/cli"
 )
 
-func listTable(c *cli.Context, many []map[string]interface{}, keys []string) {
-	w := tabwriter.NewWriter(c.App.Writer, 0, 8, 1, '\t', 0)
+func listTable(writer io.Writer, many []map[string]interface{}, keys []string) {
+	w := tabwriter.NewWriter(writer, 0, 8, 1, '\t', 0)
```

Comment:
This makes more sense to use a writer anyways and should make unit testing easier.